### PR TITLE
Feature/children update

### PR DESCRIPTION
**Pull request by leachtucker**
# Description
Updates the children's table with two new columns (_Email_, _CharacterName_).

Fixes #
- What work was done?
Created a new Knex migration. Adjust seed files to comply with the new schema.
- Why was this work done?
The prototype including two new fields for each child: Email and Character Name, but this functionality did not yet exists on the BE.
- What feature / user story is it for?
_As a parent, I want to be able to add/edit my children's profiles on the fly, so that I can make changes if anything occurs._
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

**Comment by leachtucker** [comment]
[As a parent, I want to be able to add/edit my children's profiles on the fly, so that I can make changes if anything occurs.](https://trello.com/c/nZKSDIFm/55-as-a-parent-i-want-to-be-able-to-add-edit-my-childrens-profiles-on-the-fly-so-that-i-can-make-changes-if-anything-occurs)

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "dotenv": "^8.2.0",
     "emoji-regex": "^9.2.0",
     "express": "~4.16.1",
-    "faker": "^4.1.0",
+    "faker": "^5.5.3",
```
Comment (leachtucker) [inline package.json:61]: Updated faker package to make use of new methods.

File: data/seeds/006_Children.js [hunk 2]
```diff
@@ -7,65 +7,74 @@ const children1 = [...new Array(8)].map((i, idx) => ({
   PIN: `${bc.hashSync(`0000`, process.env.BCRYPT_ROUNDS || 6)}`,
   // ParentID: `${Math.floor((idx + 4) / 4)}`,
   ParentID: `${(idx % 4) + 1}`, // 1, 2, 3,
-  AvatarID: `${faker.random.number({ min: 1, max: 10 })}`,
-  GradeLevelID: `${faker.random.number({ min: 1, max: 6 })}`,
+  AvatarID: `${faker.datatype.number({ min: 1, max: 10 })}`,
```
Comment (leachtucker) [inline data/seeds/006_Children.js:10]: Fixing some deprecated method calls here

**Review by StefxM — APPROVED** [review]
Looks amazingg

**Review by MosharrafMusa — APPROVED** [review]
looks great

**Review by StefxM — APPROVED** [review]
Looks good

**Pull request merged**